guides/common/modules/con_upgrade-paths.adoc

```diff
@@ -8,6 +8,11 @@ ifdef::satellite[]
 For more information, see {UpgradingPreviousDocURL}[_{UpgradingPreviousDocTitle}_] or {UpgradingDisconnectedPreviousDocURL}[_{UpgradingDisconnectedPreviousDocTitle}_].
 endif::[]
 
+.Hammer and API considerations
+
+If you have any scripts that use the Hammer CLI tool, ensure that you modify these scripts according to the changes in Hammer.
+If you have any integrations that use the {Project} REST API, ensure that you modify these integrations according to the changes in the API.
```

Comment:
As @evgeni suggested, it would be great if you can link to the exact section where the changes are described but at this point I don't know if we can.

Reply:
At this point we don't have it yet. We'll be able to add links later.

Reply:
@maximiliankolb I believe we meant links to release notes with description of changes of Hammer/API. I don't think that linking the Hammer or API guide as such would be particularly useful during upgrade.

Reply:
Unfortunately, the API and Hammer changes don't have a single section in our Release notes. The changes are spread over New features, Deprecated functionality, etc. So I'll add just a link to the whole RNs.